web/src/components/layout/project-data-nav.tsx

```diff
@@ -3,9 +3,11 @@ import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { useProjectPageView } from '../../hooks/stores/useProjectPageView';
 import { ViewSelector } from '../project/view-selector';
 
-type PageView = 'samples' | 'subsamples' | 'config';
+type PageView = 'samples' | 'subsamples' | 'config' | 'help';
 
-type NavProps = {};
+type NavProps = {
+  filteredSamples: string[];
```

Comment:
Just curious, is this the list of sample ids?

Reply:
yes, but only the filtered ones